apps/extension/src/ui/domains/Staking/NomPoolBond/useNomPoolBondWizard.ts

```diff
@@ -0,0 +1,359 @@
+      default:
+        return true
+    }
+  }, [claimPermission])
+
+  // we must craft a different extrinsic if the user is already staking in a pool
+  const hasJoinedNomPool = useMemo(() => !!currentPool, [currentPool])
+
+  const { data: sapi } = useScaleApi(token?.chain?.id)
+
+  const {
+    data: payloadAndMetadata,
+    isLoading: isLoadingPayload,
+    error: errorPayload,
+  } = useQuery({
+    queryKey: [
+      "getNomPoolStakingPayload",
+      sapi?.id,
+      address,
+      poolId,
+      plancks?.toString(),
+      isFormValid,
+      hasJoinedNomPool,
+      withSetClaimPermission,
+    ],
+    queryFn: async () => {
+      if (!sapi || !address || !poolId || !plancks) return null
+      if (!isFormValid) return null
+
+      return getNomPoolStakingPayload(
+        sapi,
+        address,
+        poolId,
+        plancks,
+        hasJoinedNomPool,
+        withSetClaimPermission
+      )
+    },
+    enabled: !!sapi,
+  })
+
+  const { payload, txMetadata } = payloadAndMetadata || {}
+
+  const {
+    // used to get an estimate before amount is known, for estimating maxPlancks
+    data: fakeFeeEstimate,
+  } = useQuery({
+    queryKey: [
+      "getNomPoolStakingPayload/estimateFee",
+      sapi?.id,
+      poolId,
+      address,
+      minJoinBond?.toString(),
+      hasJoinedNomPool,
+      withSetClaimPermission,
+    ],
+    queryFn: async () => {
+      if (!sapi || !address || !poolId || typeof minJoinBond !== "bigint") return null
+
+      const { payload } = await getNomPoolStakingPayload(
+        sapi,
+        address,
+        poolId,
+        minJoinBond,
+        hasJoinedNomPool,
+        withSetClaimPermission
+      )
+      return sapi.getFeeEstimate(payload)
+    },
+    enabled: !!sapi,
+  })
+
+  const {
+    data: feeEstimate,
+    isLoading: isLoadingFeeEstimate,
+    error: errorFeeEstimate,
+  } = useQuery({
+    queryKey: ["feeEstimate", sapi?.id, payload], // safe stringify because contains bigint
+    queryFn: () => {
+      if (!sapi || !payload) return null
+      return sapi.getFeeEstimate(payload)
+    },
+  })
+
+  const onSubmitted = useCallback(
+    (hash: Hex) => {
+      genericEvent("NomPool Bond", { tokenId, isBondExtra: hasJoinedNomPool })
+      if (hash) setState((prev) => ({ ...prev, step: "follow-up", hash }))
+    },
+    [genericEvent, hasJoinedNomPool, setState, tokenId]
+  )
+
+  const maxPlancks = useMemo(() => {
+    if (!balance || !existentialDeposit || !fakeFeeEstimate) return null
+    // use 11x fake fee estimate as we block form based on 10x the real fee estimate
+    if (existentialDeposit.planck + fakeFeeEstimate * 11n > balance.transferable.planck) return null
+    return balance.transferable.planck - existentialDeposit.planck - fakeFeeEstimate * 11n
+  }, [balance, existentialDeposit, fakeFeeEstimate])
+
+  const inputErrorMessage = useMemo(() => {
+    if (isSoloStaking)
+      return t("An account cannot do both regular staking and nomination pool staking")
```

Comment:
Requested by Toby: can we make the element containing this message a bit smaller to make it break more evenly? Or add a line break half way through the message?